cli-plugins/apps/how-to-guides.hbs.md

```diff
@@ -0,0 +1,611 @@
+1. In `Tiltfile`, first change the `SOURCE_IMAGE` variable to use your registry and project.
+1. At the very end of the file add:
+
+   ```bash
+   allow_k8s_contexts('your-cluster-name')
+   ```
+
+1. Inside the directory, run:
+
+   ```bash
+   tanzu apps workload apply tanzu-java-web-app --live-update --local-path . -s
+   gcr.io/my-project/tanzu-java-web-app-live-update -y
+   ```
+
+   Expected output:
+
+   ```bash
+   The files and directories listed in the .tanzuignore file are being excluded from the uploaded source code.
+   Publishing source in "." to "gcr.io/my-project/tanzu-java-web-app-live-update"...
+   📥 Published source
+   
+   🔎 Create workload:
+       1 + |---
+       2 + |apiVersion: carto.run/v1alpha1
+       3 + |kind: Workload
+       4 + |metadata:
+       5 + |  name: tanzu-java-web-app
+       6 + |  namespace: default
+       7 + |spec:
+       8 + |  params:
+       9 + |  - name: live-update
+      10 + |    value: "true"
+      11 + |  source:
+      12 + |    image: gcr.io/my-project/tanzu-java-web-app-live-update:latest@sha256:3c9fd738492a23ac532a709301fcf0c9aa2a8761b2b9347bdbab52ce9404264b
+   👍 Created workload "tanzu-java-web-app"
+
+   To see logs:   "tanzu apps workload tail tanzu-java-web-app --timestamp --since 1h"
+   To get status: "tanzu apps workload get tanzu-java-web-app"
+
+   ```
+
+1. Run Tilt to deploy the workload.
+
+    ```bash
+    tilt up
+
+    Tilt started on http://localhost:10350/
+    v0.23.6, built 2022-01-14
+
+    (space) to open the browser
+    (s) to stream logs (--stream=true)
+    (t) to open legacy terminal mode (--legacy=true)
+    (ctrl-c) to exit
+    Tilt started on http://localhost:10350/
+    v0.23.6, built 2022-01-14
+
+    Initial Build • (Tiltfile)
+    Loading Tiltfile at: /path/to/repo/tanzu-java-web-app/Tiltfile
+    Successfully loaded Tiltfile (1.500809ms)
+    tanzu-java-w… │
+    tanzu-java-w… │ Initial Build • tanzu-java-web-app
+    tanzu-java-w… │ WARNING: Live Update failed with unexpected error:
+    tanzu-java-w… │   Cannot extract live updates on this build graph structure
+    tanzu-java-w… │ Falling back to a full image build + deploy
+    tanzu-java-w… │ STEP 1/1 — Deploying
+    tanzu-java-w… │      Objects applied to cluster:
+    tanzu-java-w… │        → tanzu-java-web-app:workload
+    tanzu-java-w… │
+    tanzu-java-w… │      Step 1 - 8.87s (Deploying)
+    tanzu-java-w… │      DONE IN: 8.87s
+    tanzu-java-w… │
+    tanzu-java-w… │
+    tanzu-java-w… │ Tracking new pod rollout (tanzu-java-web-app-build-1-build-pod):
+    tanzu-java-w… │      ┊ Scheduled       - (…) Pending
+    tanzu-java-w… │      ┊ Initialized     - (…) Pending
+    tanzu-java-w… │      ┊ Ready           - (…) Pending
+    ...
+    ```
+
+<!-- add info regarding debug flag -->
+
+## <a id='export-usage'> --export usage
+
+When using `--export` flag, the user can retrieve the workload pared out of some fields such as status and metadata fileds as `creationTimestamp`.
```

Comment:
When using the `--export` flag, the user can retrieve the workload definition with all the extraneous, cluster-specific, properties/values removed (e.g. the status and metadata fileds like `creationTimestamp`) so that the workload definition could be saved and applied to a different environment without having to make significant edits.